addons/stock/tests/test_move.py

```diff
@@ -5874,23 +5874,81 @@ def test_move_line_aggregated_product_quantities_incomplete_package(self):
         backorder.action_cancel()  # Cancels the automatically created backorder.
 
         aggregate_values = picking.move_line_ids._get_aggregated_product_quantities()
-        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}']
+        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}_']
         self.assertEqual(aggregated_val['qty_ordered'], 15)
         self.assertEqual(aggregated_val['qty_done'], 5)
 
         aggregate_values = picking.move_line_ids._get_aggregated_product_quantities(strict=True)
-        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}']
+        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}_']
         self.assertEqual(aggregated_val['qty_ordered'], 5)
         self.assertEqual(aggregated_val['qty_done'], 5)
 
         aggregate_values = picking.move_line_ids._get_aggregated_product_quantities(except_package=True)
-        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}']
+        aggregated_val = aggregate_values[f'{self.product.id}_{self.product.name}__{self.product.uom_id.id}_']
         self.assertEqual(aggregated_val['qty_ordered'], 10)
         self.assertEqual(aggregated_val['qty_done'], False)
 
         self.assertEqual(self.env['stock.quant']._get_available_quantity(self.product, self.stock_location), 20)
         self.assertEqual(self.env['stock.quant']._get_available_quantity(self.product, self.customer_location), 5)
 
+    def test_move_line_aggregated_product_quantities_packagings(self):
+        """ Test the `stock.move.line` method `_get_aggregated_product_quantities`,
+        which returns data used to print delivery slips, with product packagings
+        """
+        self.env.user.groups_id += self.env.ref("product.group_stock_packaging")
+        packaging_of_4 = self.env['product.packaging'].create({
```

Comment:
To be more complete, I think this test should also use a product packaging using another UOM, so we're sure `_compute_qty` will handle correctly the conversion

Reply:
The packaging uom is `readonly` + `related = product_id.uom_id`, so I added a move with a different uom from the product uom into the test instead